packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -694,20 +694,24 @@ export class MarketOperationUtils {
                         rfqt,
                     );
                     if (firmQuotes.length > 0) {
+                        // Compute the RFQ order fillable amounts. This is done by performing a "soft" order
+                        // validation and by checking order balances that are monitored by our worker.
+                        // If a firm quote validator does not exist, then we assume that all orders are valid.
+                        const firmQuoteSignedOrders = firmQuotes.map(quote => quote.signedOrder);
+                        const rfqOrderFillableAmounts =
+                            rfqt.firmQuoteValidator === undefined
+                                ? firmQuoteSignedOrders.map(signedOrder => signedOrder.takerAssetAmount)
```

Comment:
Just to clarify:

This will be undefined until we build this out, so here we're just assuming it's 100% fillable.

Reply:
That is correct